stocks/stocks.py

```diff
@@ -1,7 +1,155 @@
-from flask import Blueprint
+import requests
+import os
+import json
+import sqlite3 as sql
+import config
 
-stocks_bp = Blueprint('stocks_bp', __name__)
+from flask import Blueprint, request
+from dotenv import load_dotenv
 
-@stocks_bp.route('/')
+
+load_dotenv()
+
+FINNHUB_TOKEN = os.getenv("FINNHUB_TOKEN")
+
+STOCK_QUOTE_URL = "https://finnhub.io/api/v1/quote?token={token}&symbol={symbol}"
+STOCK_PROFILE_URL = "https://finnhub.io/api/v1/stock/profile2?token={token}&symbol={symbol}"
+
+stocks_bp = Blueprint("stocks_bp", __name__)
+
+@stocks_bp.route("/example")
 def index():
-    return "This is an example stock app"
\ No newline at end of file
+    return "This is an example stock app"
+
+def get_stock_quote(symbol):
+    """
+    Get real-time quote data for a stock given symbol
+    """
+    r = requests.get(STOCK_QUOTE_URL.format(token=FINNHUB_TOKEN, symbol=symbol))
+    response = r.json()
+    return response
+    """
```

Comment:
Is this the response example? Perhaps move to docstring above

Also consider wrapping this in dataclass so it's less confusing and give your resp a type

Reply:
Thinking about this more, this will be nice to move into docstring because right now we have no idea what the function does (we don't know what STOCK_QUOTE_URL returns either because the response is not documented or not returned in a type that we know